2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Current SDK version is 0.15.8
2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Configure stats pid to 289868
2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/dolphin_classification/notebooks/wandb/settings
2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-14 02:40:07,241 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:_log_setup():507] Logging user logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230814_024007-by4gyeij/logs/debug.log
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:_log_setup():508] Logging internal logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230814_024007-by4gyeij/logs/debug-internal.log
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:_jupyter_setup():453] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff3235f9af0>
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:init():547] calling init triggers
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'arch': 'InceptionV4'}
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:init():596] starting backend
2023-08-14 02:40:07,242 INFO    MainThread:289868 [wandb_init.py:init():600] setting up manager
2023-08-14 02:40:07,244 INFO    MainThread:289868 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-08-14 02:40:07,246 INFO    MainThread:289868 [wandb_init.py:init():606] backend started and connected
2023-08-14 02:40:07,263 INFO    MainThread:289868 [wandb_run.py:_label_probe_notebook():1234] probe notebook
2023-08-14 02:40:07,264 INFO    MainThread:289868 [wandb_run.py:_label_probe_notebook():1244] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-08-14 02:40:07,264 INFO    MainThread:289868 [wandb_init.py:init():697] updated telemetry
2023-08-14 02:40:07,329 INFO    MainThread:289868 [wandb_init.py:init():730] communicating run to backend with 60.0 second timeout
2023-08-14 02:40:07,544 INFO    MainThread:289868 [wandb_run.py:_on_init():2180] communicating current version
2023-08-14 02:40:07,595 INFO    MainThread:289868 [wandb_run.py:_on_init():2189] got version response 
2023-08-14 02:40:07,596 INFO    MainThread:289868 [wandb_init.py:init():781] starting run threads in backend
2023-08-14 02:40:07,703 INFO    MainThread:289868 [wandb_run.py:_console_start():2159] atexit reg
2023-08-14 02:40:07,703 INFO    MainThread:289868 [wandb_run.py:_redirect():2014] redirect: wrap_raw
2023-08-14 02:40:07,704 INFO    MainThread:289868 [wandb_run.py:_redirect():2079] Wrapping output streams.
2023-08-14 02:40:07,704 INFO    MainThread:289868 [wandb_run.py:_redirect():2104] Redirects installed.
2023-08-14 02:40:07,705 INFO    MainThread:289868 [wandb_init.py:init():822] run started, returning control to user process
2023-08-14 03:07:31,542 INFO    MainThread:289868 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-08-14 03:07:31,542 INFO    MainThread:289868 [wandb_init.py:_pause_backend():418] pausing backend
2023-08-14 03:08:49,698 INFO    MainThread:289868 [wandb_init.py:_resume_backend():423] resuming backend
2023-08-14 03:08:49,704 INFO    MainThread:289868 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-08-14 03:08:49,704 INFO    MainThread:289868 [wandb_init.py:_pause_backend():418] pausing backend
2023-08-14 03:10:13,147 INFO    MainThread:289868 [wandb_init.py:_resume_backend():423] resuming backend
2023-08-14 03:10:13,152 INFO    MainThread:289868 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-08-14 03:10:13,152 INFO    MainThread:289868 [wandb_init.py:_pause_backend():418] pausing backend
2023-08-14 03:10:48,217 INFO    MainThread:289868 [wandb_init.py:_resume_backend():423] resuming backend
2023-08-14 03:10:48,220 INFO    MainThread:289868 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-08-14 03:10:48,220 INFO    MainThread:289868 [wandb_init.py:_pause_backend():418] pausing backend
2023-08-14 03:10:53,143 INFO    MainThread:289868 [wandb_init.py:_resume_backend():423] resuming backend
2023-08-14 03:10:53,729 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Current SDK version is 0.15.8
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Configure stats pid to 289868
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/dolphin_classification/notebooks/wandb/settings
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_init.py:_log_setup():507] Logging user logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230814_031053-26yven7h/logs/debug.log
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_init.py:_log_setup():508] Logging internal logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230814_031053-26yven7h/logs/debug-internal.log
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_init.py:init():547] calling init triggers
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'arch': 'InceptionV4'}
2023-08-14 03:10:53,730 INFO    MainThread:289868 [wandb_init.py:init():571] re-initializing run, found existing run on stack: by4gyeij
2023-08-14 03:10:53,732 INFO    MainThread:289868 [wandb_run.py:_finish():1894] finishing run yashchks87/dolphin/by4gyeij
2023-08-14 03:10:53,732 INFO    MainThread:289868 [jupyter.py:save_history():445] not saving jupyter history
2023-08-14 03:10:53,732 INFO    MainThread:289868 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-08-14 03:10:53,732 INFO    MainThread:289868 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-08-14 03:10:53,732 INFO    MainThread:289868 [wandb_run.py:_atexit_cleanup():2128] got exitcode: 0
2023-08-14 03:10:53,732 INFO    MainThread:289868 [wandb_run.py:_restore():2111] restore
2023-08-14 03:10:53,732 INFO    MainThread:289868 [wandb_run.py:_restore():2117] restore done
